pkg/kubelet/custommetrics/custom_metrics.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package custommetrics contains support for instrumenting cAdvisor to gather custom metrics from pods.
+package custommetrics
+
+import (
+	"path"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	CustomMetricsDefinitionContainerFile = "definition.json"
+
+	CustomMetricsDefinitionDir = "/etc/custom-metrics"
```

Comment:
Seems like this goes into /etc/cadvisor?

On Tue, Feb 2, 2016 at 7:17 AM, Marcin Wielgus <notifications@github.com>
wrote:

> In pkg/kubelet/custommetrics/custom_metrics.go
> <https://github.com/kubernetes/kubernetes/pull/19944#discussion_r51581225>
> :
>
> > +limitations under the License.
> > +*/
> > +
> > +// Package custommetrics contains support for instrumenting cAdvisor to gather custom metrics from pods.
> > +package custommetrics
> > +
> > +import (
> > +	"path"
> > +
> > +	"k8s.io/kubernetes/pkg/api"
> > +)
> > +
> > +const (
> > +	CustomMetricsDefinitionContainerFile = "definition.json"
> > +
> > +	CustomMetricsDefinitionDir = "/etc/custom-metrics"
>
> Yes, I don't want put CM into the default kubernetes-related config.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/19944/files#r51581225>.
>